src/modules/collections/services/collections.service.ts

```diff
@@ -105,6 +121,9 @@ export class CollectionsService {
 			createdAt: gqlCollection.created_at,
 			updatedAt: gqlCollection.updated_at,
 			isDefault: gqlCollection.is_default,
+			usedForLimitedAccessUntil:
+				this.adaptByVisitEndAt(await this.visitsService.findByFolderId(gqlCollection.id)) ||
```

Comment:
If findByFolderId throws an error, this whole function will stop and the user will not see anything.
The usedForLimitedAccessUntil is only an informative low priority thing for the user to know. So if this fails, we should still return the rest of the folder info.

So we might want to move this to before the return and wrap it in a try catch. In the catch we can log the error to the console (console.error()). That way if it fails, it will just return undefined and not crash the whole folders call.
```
usedForLimitedAccessUntil: undefined
```

We could also do the try catch inside the findByFolderId function. But i think it is better wrapped here. Since we might want to use that function elsewhere, where we do expect an error if something goes wrong.

Reply:
Adjusted